frontend/relic-table.tsx

```diff
@@ -81,6 +84,19 @@ export const RelicTable = () => {
 
     return (
         <Box a="center">
+            <Breadcrumb fontWeight='medium' fontSize='sm'>
+                <BreadcrumbItem>
+                    <BreadcrumbLink href='/'>Home</BreadcrumbLink>
+                </BreadcrumbItem>
+
+                <BreadcrumbItem>
+                    <BreadcrumbLink href='/dashboard'>Dashboard</BreadcrumbLink>
+                </BreadcrumbItem>
+
+                <BreadcrumbItem isCurrentPage>
+                    <BreadcrumbLink href="/dashboard/">Relics</BreadcrumbLink>
```

Comment:
this looks like its going to the wrong place to me? Or is this final item "dead" anyway?